test/unittests/api/v0.0.1/candidates/Visit_Test.php

```diff
@@ -89,6 +89,7 @@ function testPutInvalidBattery() {
         $this->assertEquals($API->Headers, ["HTTP/1.1 400 Bad Request"]);
     }
     function testPostValidVisit() {
+        \User::singleton("admin");
```

Comment:
This looks like dead code, or at least code that's depending on side-effects when it shouldn't.